setup.py

```diff
@@ -1,31 +1,29 @@
-import setuptools, sys, subprocess
+import setuptools
 
-try:
-    output = subprocess.run([sys.executable, '-m', 'pip', 'install','-r','requirements.txt'])
-except:
-    output = subprocess.run(['pip', 'install','-r','requirements.txt'])
```

Comment:
`install_requires` allows automatic handling of dependencies during installation.

[install-requires-vs-requirements-files](https://packaging.python.org/discussions/install-requires-vs-requirements/#install-requires-vs-requirements-files)